css/style.css

```diff
@@ -288,6 +332,20 @@ iframe {
 		text-decoration: underline;
 	}
 
+	section.details div div {
```

Comment:
here is the issue to with your contact/links section being white.
When you removed class `clearfix` from [here](https://github.com/learn-co-students/hs-zhw-shoes-layout-v-000/pull/7/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR180) - you removed some of the styles that came with it. I'd suggest adding `class=clearfix` and try to figure out why footer changed color as well